### PR TITLE
Adding simple splash screen.

### PR DESCRIPTION
**Pull request by proyectox123**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,21 +9,26 @@
...
             <intent-filter>
-                <action android:name="android.intent.action.VIEW" />
                 <action android:name="android.intent.action.MAIN" />
-
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name="com.mho.training.features.main.MainActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
```
Comment (jesusdmedinac) [inline app/src/main/AndroidManifest.xml:29]: What this implementation do?

File: app/src/main/java/com/mho/training/features/splash/SplashActivity.kt [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import com.mho.training.utils.startActivity
+
+class SplashActivity : AppCompatActivity() {
+
+    //region Override Methods & Callbacks
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        startActivity<MainActivity> {}
```
Comment (jesusdmedinac) [inline app/src/main/java/com/mho/training/features/splash/SplashActivity.kt:15]: Does startActivity wait some seconds to start MainActivity?

**Review by jesusdmedinac — COMMENTED** [review]
I have only some questions.

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (proyectox123) [reply app/src/main/AndroidManifest.xml:29]: SplashActivity is now the launcher activity, so MainActivity keeps the action view. Also, the compilator throws:

_App is not indexable by Google Search;  At least one Activity with an ACTION-VIEW intent-filler._

File: app/src/main/java/com/mho/training/features/splash/SplashActivity.kt (hunk 2 above)
Comment (proyectox123) [reply app/src/main/java/com/mho/training/features/splash/SplashActivity.kt:15]: The splash lasts until the mainly activity is ready.

https://medium.com/@sneyderangulo/como-implementar-splash-screen-correctamente-en-android-f6abcc592b4c

**Pull request merged**